code/src/index.css

```diff
@@ -11,3 +11,106 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main {
+  font-family: "Montserrat", sans-serif;
+  height: 100%;
+  background: #b65fdf;
+  background: -webkit-linear-gradient(top right, #b65fdf, #d88f80);
+  background: -moz-linear-gradient(top right, #b65fdf, #d88f80);
+  background: linear-gradient(to bottom left, #b65fdf, #d88f80);
+}
+
+header {
+  padding: 50px;
+}
+
+h1 {
+  text-align: center;
+  margin: 0;
+}
+
+.loading {
+  text-align: center;
+}
+
+.new-thought-container {
+  background: #d9dce8;
+  background: -webkit-linear-gradient(top left, #d9dce8, #f0bdee);
+  background: -moz-linear-gradient(top left, #d9dce8, #f0bdee);
+  background: linear-gradient(to bottom right, #d9dce8, #f0bdee);
+}
+
+.new-thought-form {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 30px;
```

Comment:
I need to use grid gap more, I go for margin way to often. This seems to be the best approach!